DEPLOYMENT.md

```diff
@@ -0,0 +1,28 @@
+# Pinning server deployment
+
+The pinning server is a node.js process that is running using a the PM2 process manager.
+
+## Deployment
+
+Server deployment is manual at the moment. Continuous deployment is gonna be added in the future.
+
+To deploy it you should enter `ssh` into the machine, and then `cd` into `3box-pinning-server`
```

Comment:
Don't you also need to do a git pull in that dir?

Reply:
The script handles that

Reply:
ok, cool.